setup.py

```diff
@@ -142,7 +142,7 @@
     "pymedphys.tests.e2e": [
         "cypress/*",
         "cypress/archive/*",
-        "cypress/fixtures/.gitignore",
+        "cypress/fixtures/*",
```

Comment:
`setup.py` is autogenerated. I suspect it will be some combination of which platform `poetry run pymedphys dev propagate` was run on + what version of poetry is installed.